docs/README.md

```diff
@@ -74,20 +77,21 @@ The baseset directories are:
 The `/baseset/` directory inside of the OpenTTD installation can also be used.
 4. Run OpenTTD. Chances are that you'll miss a sound set. Get one (we recommend our sister project OpenSFX) and install it into the same directory as OpenGFX.
 5. In the main menu of the game, click the Game Options button. The Game Options dialog will appear.
-6. Select OpenGFX from the drop-down list below Base graphics set if that's not selected already (bottom left of window).  Close the window using the × in the upper left corner.
+6. Select OpenGFX from the drop-down list below Base graphics set if that's not selected already (bottom left of window). Close the window using the × in the upper left corner.
 - If you did not install the original TTD base graphics during the installation of OpenTTD, you can skip this step.
 - If you installed the original TTD base graphics as well, this is where you can switch base graphic sets.
 
 Now that wasn't so hard, was it?
 
-Anyway, if you're having trouble getting OpenGFX to work, please file a detailed report on what you did, what error messages you got and where you got stuck in the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162) or (preferably) at our [issue tracker](https://github.com/OpenTTD/OpenGFX/issues).
+Anyway, if you're having trouble getting OpenGFX to work, please file a detailed report on what you did, what error messages you got and where you got stuck in the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162) or (preferably) at our [issue tracker](https://dev.openttdcoop.org/projects/opengfx/issues) on the http://dev.openttdcoop.org website.
+
 
 ## 4.0 Reporting bugs and contributing
 
 ### 4.1 Reporting bugs
 
-If you spot any graphical bugs or glitches in the available graphics, please let us know preferrably via our [issue tracker](https://github.com/OpenTTD/OpenGFX/issues) or via the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162).
-Please make sure that you're using the latest available version before reporting a bug. You can check the [issue tracker](https://github.com/OpenTTD/OpenGFX/issues) to see if the bug you've found is already reported (or fixed!).
+If you spot any graphical bugs or glitches in the available graphics, please let us know preferrably via our [issue tracker](https://dev.openttdcoop.org/projects/opengfx/issues) or via the OpenGFX release topic on the [TT-Forums](http://www.tt-forums.net/viewtopic.php?f=36&t=40162).
```

Comment:
The issue tracker etc probably should now link to GitHub as well. If we move the project, we move it completely

Reply:
I wanted to use the GitHub issue tracker initially, though due to the issue tracker not being set up on this repo I decided to put the openttdcoop.org's issue tracker in. I can sure change this, but currently visiting https://github.com/OpenTTD/OpenGFX/issues redirects to https://github.com/OpenTTD/OpenGFX/pulls.

Reply:
Oh. That actually is an oversight on my part. Let's quickly activate the issues. Thanks for that pointer!

Reply:
Also mentioning a site which can also be moved from the openttcoop.org's site to GitHub:
There is a Wiki section available in a GitHub repository, which could be activated. 👍

Reply:
Yes... what purpose would this wiki serve, though? Anything more general baseset probably is better placed in the OpenTTD wiki